assets/alerts.html

```diff
@@ -17,31 +17,47 @@
   </script>
 </head>
 <body>
-  <div id="alertbox">
+  <div id="subscriptionBox">
     <b class="username"></b> acabou de se inscrever!<br />
     Seja bem vindo(a) 😎
   </div>
 
+  <div id="resubBox">
+    Obrigado pelo resub de <b class="months"></b> mes(es) <b class="username"></b>
```

Comment:
Seria legal colocar `streakMonths` aqui também e acho que como é resub, nunca vai ser `mes`, sempre serão `meses`.

```suggestion
    Obrigado pelo resub de <b class="streakMonths"></b> meses <b class="username"></b>
```